inlang/source-code/paraglide/paraglide-js/README.md

```diff
@@ -147,6 +147,15 @@ import * as m from "./paraglide/messages"
 const msg = m.hello({ name: "Samuel" }, { languageTag: "de" }) // Hallo Samuel!
 ```
 
+## Usage with a Bundler
+
+We provide a few bundler plugins to make it easier to use paraglide-js with a bundler. If you
+are using one of these bundlers, we recommed using the corresponding plugin:
+
+- [Rollup](https://github.com/inlang/monorepo/tree/main/inlang/source-code/paraglide/paraglide-js-adapter-rollup)
```

Comment:
Maybe we could add some key words about what these plugins are good for.

Reply:
Will do!